src/common/SPELLS/classic/warrior.ts

```diff
@@ -37,22 +36,14 @@ const spells = {
     icon: 'ability_bullrush',
   },
   CHARGE: {
-    id: 11578,
+    id: 100,
     name: 'Charge',
     icon: 'ability_warrior_charge',
```

Comment:
I've been adding `.jpg` to the end of all icons in case it's missing from the place it gets pulled from.